operator/redisfailover/handler.go

```diff
@@ -52,15 +50,15 @@ func NewRedisFailoverHandler(config Config, rfService rfservice.RedisFailoverCli
 		mClient:    mClient,
 		k8sservice: k8sservice,
 		logger:     logger,
-		labels:     labels,
+		labels:     defaultLabels,
```

Comment:
If the labels are not configurable anymore and are managed (generate, merge...) at runtime, do we need this field if always are the `defaultLabels` and the field is not mutated in any part of the struct?

Reply:
Changed. You're absolutely right. I forgot about it on the labels change. Thanks!